front/components/assistant/conversation/input_bar/editor/extensions/ParagraphExtension.ts

```diff
@@ -3,8 +3,6 @@ import { Paragraph } from "@tiptap/extension-paragraph";
 export const ParagraphExtension = Paragraph.extend({
   addKeyboardShortcuts() {
     return {
-      ...this.parent?.(),
```

Comment:
probably super minor but we don't wanna keep the existing shortcuts? (which is [setParagraph()](https://tiptap.dev/docs/editor/extensions/nodes/paragraph#keyboard-shortcuts))